yb-voyager/src/dbzm/config.go

```diff
@@ -139,3 +142,38 @@ func (c *Config) WriteToFile(filePath string) error {
 	}
 	return nil
 }
+
+// read config file DEBEZIUM_CONF_FILEPATH into a string
+func readConfigFile() (string, error) {
+	configFile, err := os.ReadFile(DEBEZIUM_CONF_FILEPATH)
+	if err != nil {
+		return "", fmt.Errorf("failed to read config file %s: %w", DEBEZIUM_CONF_FILEPATH, err)
+	}
+
+	return string(configFile), nil
+}
+
+// generate/fetch the value for 'debezium.source.database.server.id' property for MySQL
+func getDatabaseServerID() int {
+	databaseServerId := rand.Intn(100000)
```

Comment:
Ensure that the id is greater than 10000. Just to avoid any conflict with existing IDs.